### DISCUSSION TITLE
Request limits and other data

### DISCUSSION THREAD

**Discussion by RemiP12** [ip-tools/python-epo-ops-client#53] [Q&A] [answered]
Hello, thank you very much for your library!
I have 2 questions:

1. According to their Fair Use Charter, the EPO limits:
     - human queries to 10/min,
     - automated queries via the REST API to 5GB/7 days,
     - automated queries via OPS to 4GB/week.
     Which limit applies to your epo-ops client?
2. Can your client be used to obtain other data than the bibliographic data? For example, the legal status or claims?

Thank you!

**Comment by gsong** [comment, marked as answer]
The library observes the rate limit from OPS. Supported queries are documented: https://github.com/gsong/python-epo-ops-client#client
